apps/gnome-mobile/src/lib/api/Gnomes.service.ts

```diff
@@ -13,7 +13,17 @@ const getMyGnomesInteractions = async () => {
     .then((response) => response.data);
 };
 
+const getNearestGnomes = async (id: string): Promise<Gnome[]> => {
+  return await axiosInstance
+    .get(`api/rest/v1/gnomes/${id}?limit=3`)
```

Comment:
to limit = 3 nic nie robi chyba
```suggestion
    .get(`api/rest/v1/gnomes/${id}`)
```